src/main/java/edu/ucsb/ucsbcslas/controllers/ActiveQuarterController.java

```diff
@@ -69,6 +69,26 @@
 
     List<ActiveQuarter> activeQList = activeQuarterRepo.findAll();
     ObjectMapper mapper = new ObjectMapper();
+    String activeQuarter = activeQList.get(0).getActiveQuarter();
+    if(activeQuarter.length() == 5) {
+      String quarter = activeQuarter.substring(4);
+      String year = activeQuarter.substring(2,4);
+      if(quarter.equals("1")){
+        quarter = "W";
+      }
+      else if(quarter.equals("2")){
+        quarter = "S";
+      }
+      else if(quarter.equals("3")){
+        quarter = "M";
+      }
+      else {
+        quarter = "F";
+      }
+      activeQList.get(0).setActiveQuarter(quarter+year);
+    } else {
+      activeQList.get(0).setActiveQuarter(Character.toUpperCase(activeQuarter.charAt(0))+activeQuarter.substring(1));
+    }
```

Comment:
I wonder if it would be better to factor this out into a static method that you can call:

```
      activeQuarter = ActiveQuarterController.fixQuarter(activeQuarter);
      activeQList.get(0).setActiveQuarter(activeQuarter);
```

The method
```
public static String fixActiveQuarter(String activeQuarter) {
   // put code from lines 73 through 91 here, but on line 88, use
   return quarter+year
  // and on line 90 use:
  return Character.toUpperCase(activeQuarter.charAt(0))+activeQuarter.substring(1)
}
...


That way you can dry up the copy/pasted code that is duplicated in CourseController.java